enterprise/reporting/mediator.go

```diff
@@ -23,6 +23,7 @@ type Mediator struct {
 	ctx       context.Context
 	cancel    context.CancelFunc
 	reporters []types.Reporting
+	stats     stats.Stats
```

Comment:
We can probably skip setting in the struct since it's being used in the constructor only. wdyt?